unittests/mocks.py

```diff
@@ -34,3 +37,13 @@ def label(self):
 
     def __repr__(self):
         return "Population {}".format(self._label)
+
+    @property
+    def _vertex(self):
+        return self._mock_vertex
+
+
+class MockVertex(object):
```

Comment:
Why not use pacman_test_objects/simple_test_vertex.py and extend that with the extra method

optional!

Reply:
Could have... but then I was changing sPyNNaker here!